src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -440,7 +440,7 @@ public bool IsBrokerInstalledAndInvokable(AuthorityType authorityType)
                 _logger.Info("[WAM Broker] MsalRuntime init failed...");
                 _logger.InfoPii(s_initException);
 
-                return false;
+                throw new MsalServiceException("wam_runtime_init_failed", "MsalRuntime init failed.");
```

Comment:
This should be MsalClientException. 
Also please add a descriptive and actionable message. How can the user fix this error? If the solution is verbose, then maybe adding a aka.ms link to WAM wiki would be good (and add a Troubleshooting section there, if we already don't have it).

Reply:
Added https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/wam#troubleshooting

Reply:
And we now throw the MSALRuntimeException that provides the details of the error